linode_api4/linode_client.py

```diff
@@ -1618,6 +1619,69 @@ def image_create(self, disk, label=None, description=None):
 
         return Image(self, result['id'], result)
 
+    def image_create_upload(self, label: str, region: str, description: str=None) -> Tuple[Image, str]:
+        """
+        Creates a new Image and returns the corresponding upload URL.
+        https://www.linode.com/docs/api/images/#image-upload
+
+        :param label: The label of the Image to create.
+        :type label: str
+        :param region: The region to upload to. Once the image has been created, it can be used in any region.
+        :type region: str
+        :param description: The description for the new Image.
+        :type description: str
+
+        :returns: A tuple containing the new image and the image upload URL.
+        :rtype: (Image, str)
+        """
+        params = {
+            "label": label,
+            "region": region
+        }
+
+        if description is not None:
```

Comment:
Since description is optional, will the API skip updating the description when it's None in the request, or we have to check it here? I'm thinking if the API backend can do the job, we can simplify our code here to always pass the description value even when it's None.

Reply:
Hmm, I'll double check on this. It might be useful for us to make a helper function to dynamically filter out `None` keys